src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let acc = initialValue;
+    let startAtIndex = 0;
+
+    if (arguments.length < 2) {
+      startAtIndex = 1;
+      acc = this[0];
```

Comment:
What is `acc` ? Your variable names should be meaningful. Better rename to `accumulator` as said in [method](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)